woocommerce/woocommerce-ios
# Customize WordPressAuthenticator login instructions

Closes #472. Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/36.

## To Test
1. Check out this branch.
2. Clean the project.
3. Run `pod install` / `pod update`.
4. Build the project.
5. Go through the login flow. The email login screen should say "Log in with your WordPress.com account email address to manage your WooCommerce stores."

**Also in this PR:** the custom instructions that will be needed for self-hosted, jetpack-connected sites that log in.

<img src="https://user-images.githubusercontent.com/1062444/49471197-eced4b80-f7d1-11e8-8a17-bc1bed867182.png" width="350" />
